prebid-mobile/prebid-mobile.md

```diff
@@ -11,31 +11,93 @@ nav_section: prebid-mobile
 <div class="bs-docs-section" markdown="1">
 
 # Prebid Mobile Overview
-{:.no_toc}
 
-Prebid Mobile provides libraries for Android and iOS that provide an end-to-end open-source header bidding solution for mobile app publishers.
+* TOC
+{:toc }
 
-Benefits and features include:
+Prebid Mobile helps app publishers by making the advantages of “traditional” web header bidding available to mobile app environments. This is accomplished by providing libraries that enable mobile app publishers to implement end-to-end, open source header bidding solutions.
```

Comment:
Let's try to clean this up a bit as it is the first line on our overview page.  We should be very crisp about the value prop for Prebid Mobile.